src/haddock/libs/libutil.py

```diff
@@ -281,6 +281,16 @@ def get_number_from_path_stem(path):
     return int(number)
 
 
+def get_number_of_digits(num):
+    """
+    Get the number of digits of a number.
+
+    10 has two digits.
+    100 has three digits.
+    """
+    return len(str(num))
```

Comment:
`log10(num+1)` also gives this result, and 'should' be faster (although that does not matter here). But I feel like a numerical problem should not be solved by using strings ;-)

Reply:
n1!